### ISSUE TITLE
Pagination for report list aincident list

### ISSUE THREAD

**Issue by ARSSANTO** [BubbleDK/bub-mdt#27]
### The problem

Pagination might help to reduce open mdt load for big servers

### Ideal solution

Implement Pagination on Report list and incident list or where its needed

**Comment by BubbleDK** [comment]
Yeah, that's a work in progress. I'm slowing but surely getting lists and such paginated, to optimize the performance.
